canonical-docs/ubuntu-core-docs
# Update build scripts

- Change the build (not clean) scripts to use `&&` so it fails correctly if the command
errors.
- Encapsulate the bash arguments to avoid possible errors.
- _Carefully_ work out correct manifest URL from git remotes


## QA
- `./run clean` - If this isn't a fresh checkout
- `./run build` - Should build successfully
- `./run clean` again
